godtools/App/Features/LessonFilter/Domain/Interface/StoreUserLessonFiltersRepositoryInterface.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  StoreUserLessonFiltersRepositoryInterface.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 7/8/24.
+//  Copyright © 2024 Cru. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+protocol StoreUserLessonFiltersRepositoryInterface {
+    
+    func storeUserLanguageFilterPublisher(with id: String) -> AnyPublisher<Void, Never>
```

Comment:
This makes sense to me!  I go back and forth in cases like this on whether to pass an entire object vs the one attribute needed.  I guess my next question would be, how far do you pass the LessonLanguageFilterDomainModel-- like in `storeUserLessonLanguageFilter` in `UserLessonFiltersRepository`, would it make sense to inject a `UserLessonLanguageFilterDataModel`, since that's what that class returns from the getter?  Just have to map LessonLanguageFilterDomainModel to UserLessonLanguageFilterDataModel.